### DISCUSSION TITLE
Provide related variables to display meta-information

### DISCUSSION THREAD

**Discussion by howie6879** [alibaba/GraphScope#41] [Ideas]
Does GraphScope need to provide related variables to display meta-information, such as:

```ptyhon
import graphscope

print(graphscope.__version__)
# out
# 0.1

print(graphscope.__ description__)
# out
# GraphScope: A One-Stop Large-Scale Graph Computing System from Alibaba
```

we can create a script named `__version__.py` to manage this info:

```ptyhon
__name__ = 'graphscope'
__description__ = 'GraphScope: A One-Stop Large-Scale Graph Computing System from Alibaba.'
__url__ = 'https://github.com/alibaba/GraphScope'
__version__ = '0.1'
__license__ = 'Apache 2.0'
...
```
Then automatically read in setup.py.

**Comment by howie6879** [comment]
anyone there? @sighingnow @lidongze0629

**Comment by sighingnow** [comment]
Will be fixed at the next release.

**Reply by howie6879** [reply]
> Will be fixed at the next release.

When I solved #37 last night, I realized this idea by the way. If you think you need this feature, I can submit a PR

**Reply by siyuan0322** [reply]
To enable something like `graphscope.__version__`, one need to add a line in `graphscope/__init__.py` like this:
`__version__ = 0.1`.
Metadata in `setup.py` can be inspected by `importlib.metadata`.

**Comment by sighingnow** [comment]
The version tag will be decided by our release cycle management. Including version config in CMakeLists.txt, in setup.py, in possible `__version__` file, in docker version tag, and possible in CI environment settings.

**Reply by howie6879** [reply]
I mean the python SDK version of GS

**Reply by sighingnow** [reply]
GraphScope will have a tagged version as well, which, needs to be consistent with version of the python SDK.
